src/main/java/hudson/scm/listtagsparameter/ListSubversionTagsParameterDefinition.java

```diff
@@ -255,6 +256,7 @@ private boolean isSVNRepositoryProjectRoot(SVNRepository repo) {
         return true;
       }
     } catch (SVNException e) {
+      LOGGER.log(Level.SEVERE, "An SVN exception occurred", e);
```

Comment:
It's doesn't?  It tells a sysadmin that they have a misconfigured repository line when they specify svn+foo://domain/path and foo is not a configured tunnel in .subversion/config.  It's much better then throwing that exception on the floor.  It's the only way I would know how to fix the bug.